src/evalfunc.c

```diff
@@ -744,6 +753,20 @@ arg_string_or_func(type_T *type, type_T *decl_type UNUSED, argcontext_T *context
     return FAIL;
 }
 
+/*
+ * Check "type" is a list of 'any' or a class.
+ */
+    static int
+arg_class_or_list(type_T *type, type_T *decl_type UNUSED, argcontext_T *context)
+{
+    if (type->tt_type == VAR_CLASS
+	    || type->tt_type == VAR_LIST
+	    || type_any_or_unknown(type))
+	return OK;
+    arg_type_mismatch(&t_class, type, context->arg_idx + 1);
```

Comment:
Can you find out why some of these lines are not covered by the tests?

Reply:
I guess it's because there's no test with a `def` function, I couldn't get the testing harness to collaborate (I guess the `class` must be declared outside of the function itself) and I got tired.